app/views/shared/_upcomingevents.html.erb

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap -->
+    <link href="css/bootstrap.min.css" rel="stylesheet">
+
+  </head>
+  <body>
+
+    <div class="container">
+
+      <h1>
+        Upcoming Events!
+      </h1>
+
+      <div class="row">
+        <% @events.each do |event| %>
+        <div class="col-md-4">
+          <div class="card-block">
+            <img src="https://www.codaisseur.com/assets/rocket-0e991969b204c33b21331483b1f7d49499fd3ddb507ae46d6d22a658a7a3b852.png" height=50/>
```

Comment:
We learnt other ways to add an image to your projects, instead of taking them from the internet...
Do you remember? 😄